### DISCUSSION TITLE
Which port should I specify for telnet?

### DISCUSSION THREAD

**Discussion by dj-nsk** [Arkady23/V380-Web#67] [Q&A]
Добрый день! В инструкции не указано и во всех обсуждениях не нашёл: какой порт при подключении по Телнет указывать?

**Comment by dj-nsk** [comment]
Видимо, на моей камере (как и на некоторых других) telnet вообще не запускается:
AppKD_WF_V1.0.5.4_20230601
Hw_HsAkQQWG_WIFI_COMM_20221220

**Comment by Shinchan1215** [comment]
On mine too it does not start 
version HwV380E11_WF3_AKQ8_LIGHT_20180628
